planning/behavior_velocity_planner/src/scene_module/detection_area/debug.cpp

```diff
@@ -138,13 +147,28 @@ visualization_msgs::msg::MarkerArray DetectionAreaModule::createDebugMarkerArray
   const rclcpp::Time now = clock_->now();
 
   if (!debug_data_.stop_poses.empty()) {
+    MarkerColor marker_color;
+    marker_color.r = 1.0;
+    marker_color.g = 0.0;
+    marker_color.b = 0.0;
+    marker_color.a = 1.0;
```

Comment:
If you want it to be the same as it was before the change
```suggestion
    MarkerColor marker_color;
    marker_color.r = 0.1;
    marker_color.g = 0.1;
    marker_color.b = 1.0;
    marker_color.a = 0.5;
```

Reply:
Hi @yukkysaito : The requirement was to change colours of detection area(debug markers) based on obstacle object placement towards ego-vehicle.

please refer below:
Expected behavior

I can decide whether obstacles are not detected, module is unregistered or ego vehicle goes over the dead line from debug markers.

My proposal is

    module is registered but not detected: detection area marker is green
    detected: detection area marker is red
    go over the dead line: detection area marker is grey
    unregistered: delete

so we have changed code to show different colors for debug marker based on obstacle detection position of ego vehicle. Please let us know if you don't want red colour to be used when detection happens

Reply:
@raminterpl 
Thank you.
your proposal is seemed to be good. :+1: